utils.go

```diff
@@ -97,7 +97,6 @@ func readFromFile(filepath string) (s string, r []byte) {
 		//log.Println("Error reading from file:", fileError)
 	}
 	_fileContents = string(byteValue)
-
```

Comment:
Let's not make any changes in the file that is not being touched by the current use-case.